src/pumps/GeoEmaAndCumSmaPump.sol

```diff
@@ -240,7 +245,7 @@ contract GeoEmaAndCumSmaPump is IPump, IInstantaneousPump, ICumulativePump {
     function _readCumulativeReserves(address well) internal view returns (bytes16[] memory cumulativeReserves) {
         bytes32 slot = getSlotForAddress(well);
         (uint8 n, uint40 lastTimestamp, bytes16[] memory lastReserves) = slot.readLastReserves();
-        uint offset = getSlotsOffset(n) * 2;
+        uint offset = getSlotsOffset(n) << 2;
```

Comment:
missed this, this should be shifted by 1

Reply:
Good catch, fixed